### PR TITLE
Use placeholder for info of private bundle 

### PR DESCRIPTION
**Pull request by kashizui**
@percyliang @pbhatnagar3

### CODE REVIEW

File: test-cli.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def __exit__(self, exc_type, exc_value, tb):
             run_command([cl, 'wrm', '--force', worksheet])
 
         # Delete all bundles (dedup first)
-        if len(self.bundles) > 0:
+        if len(set(self.bundles)) > 0:
```
Comment (percyliang) [inline test-cli.py]: why set?

**Comment by percyliang** [comment]
This makes me a bit nervous - probably good to double check everything (does the website have to do know about private bundles too now?) - but LGTM.

File: test-cli.py (hunk 1 above)
Comment (kashizui) [inline test-cli.py]: ..not actually necessary, you're right.

**Comment by kashizui** [comment]
Tested the website, private bundles don't crash the website (see screenshot). The `PrivateBundle` subclass is just an easy way to tell the rest of the codebase that a private bundle doesn't have any metadata fields.

There are definitely better ways to do this, but until I can be more confident about how we'd want to refactor a lot of things, I think this is the smoothest solution. I do feel like I'm being significantly hampered by how the codebase is now (for example, permissions checks are only applied when one remembers to apply them, rather than there being just a few checkpoints that guarantee permission safety), and hopefully I'll be able to address these soon.

screen shot 2015-11-07 at 5 33 14 pm

**Comment by kashizui** [comment]
This fixes #202 
and also fixes codalab/codalab#1371

**Comment by percyliang** [comment]
Ok, sounds good!

**Pull request merged**